h2/src/main/org/h2/command/dml/Delete.java

```diff
@@ -30,30 +30,35 @@
 public class Delete extends Prepared {
 
     private Expression condition;
-    private TableFilter tableFilter;
+    private TableFilter targetTableFilter;
```

Comment:
please add a comment like // for MERGE..USING

Reply:
done